keep/providers/slack_provider/slack_provider.py

```diff
@@ -215,6 +216,7 @@ def _notify(
                 raise ProviderException(
                     f"{self.__class__.__name__} failed to notify alert message to Slack: {response.text}"
                 )
+            notify_data = {"slack_timestamp": utcnowtimestamp()}
```

Comment:
does that work? :O you can comment on a thread with webhook url?

Reply:
Nope, but we need it to have the results of the Slack step with webhook integration. Otherwise, we can't enrich incident/alert